Makefile

```diff
@@ -13,7 +13,7 @@ deps-update:
 lint:
 	pipenv run mypy $(packages) --disallow-untyped-defs --ignore-missing-imports --warn-unused-ignores || true # TODO(jack) Figure out why mypy is failing on 'has no attribute' error
 	pipenv run bandit -r $(packages)
-	pipenv run pylint $(packages) --disable=missing-docstring,bad-continuation,duplicate-code,W0511,R0912 --max-line-length=100
+	pipenv run pylint $(packages) --disable=missing-docstring,bad-continuation,duplicate-code,W0511,R0912,too-many-lines --max-line-length=100
```

Comment:
Do we need to add `too-many-lines` here if there is a pylint exception comment on line 786 already?

Reply:
Ahh yea, this is for the fact it doesn't like that `main.py` is so long. On 786, it is for the particular method having `too-many-statements`. 

I do like to break up methods that get too long, but the one its complaining about is`setup_parser` - unsure how to break that one up logically.  And for `main.py` - I don't know how/if we want to break it out into more modules?